parts/xmem_nozzle.go

```diff
@@ -615,14 +640,14 @@ type XmemNozzle struct {
 
 	//remote cluster reference for retrieving up to date remote cluster reference
 	remoteClusterSvc  service_def.RemoteClusterSvc
-	sourceClusterUuid string // used to calculate sourceClusterId for custom CR
 	targetClusterUuid string
+	sourceBucketUuid  string // used in HLV
 	targetBucketUuid  string
 
 	// Source and target cluster ID. Used for custom CR in XATTR _xdcr to identify source of changes.
 	// For DP, they are the cluster UUID encoded in base64. For test we can set it to any unique string.
-	sourceClusterId hlv.DocumentSourceId
-	targetClusterId hlv.DocumentSourceId
+	sourceBucketId hlv.DocumentSourceId
```

Comment:
How are these different from `sourceBucketUuid` and `targetBucketUuid` on line 644 and 645?